processor/resourcedetectionprocessor/README.md

```diff
@@ -56,11 +57,11 @@ ec2:
 
 * Amazon ECS: Queries the [Task Metadata Endpoint](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-metadata-endpoint.html) (TMDE) to record information about the current ECS Task. Only TMDE V4 and V3 are supported.
 
-    * cloud.provider (aws)
+    * cloud.provider ("aws")
     * cloud.account.id
     * cloud.region
     * cloud.zone
-    * cloud.infrastructure_service (ECS)
+    * cloud.infrastructure_service ("aws_ecs")
```

Comment:
Minor nit: can this be moved up under `cloud.provider` to have a consistent ordering of attributes with the other detectors?